README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/Erbenos/pcp-statsd-c.svg?branch=master)](https://travis-ci.com/Erbenos/pcp-statsd-c)
+[![Build Status](https://travis-ci.com/Erbenos/pcp-statsd-c.svg?branch=aggregating-values)](https://travis-ci.com/Erbenos/pcp-statsd-c)
```

Comment:
I think the best idea would be to keep README as is (on master) and configure Travis with PR testing. It can be easily set up to test PRs.

Reply:
What is wrong with current setup? The PRs are build by Travis already. The reason for the change in README.md is that before I create PR I make several commits which I want to be tested for build by Travis.

Reply:
Usually the build status in README is kept against "master" however you can configure Travis to automatically pick all PRs and test them. You need to install github hook I think.